root_VS2010/programs/C#/Frameworks/Infrastructure/ServiceInterface/ASPNETWebService/web.Config

```diff
@@ -10,6 +10,19 @@
 <configuration>
   <!-- ajaxの設定（バージョンなどに注意する。） -->
   <!-- appSettingsの設定 -->
+  <configSections>
```

Comment:
@daisukenishino 

There were not these settings until now.
Because TargetFrameworkVersion becomes "3.5", are these settings necessary?